### PR TITLE
Add/move around mainline unit portraits

### PR DESCRIPTION
**Pull request by nemaara**
For #4020. Right now I've moved the Fire Dragon portrait from TRoW and SoF to core and added the proposed water serpent portrait. I'll move around some of the orc portraits in this PR later on.

### CODE REVIEW

**Comment by nemaara** [comment]
@Lordbob I'd like to use the 6th grunt portrait variation as the new Orcish Leader portrait and move the current Leader portraits to the Orcish Ruler instead. Is that okay with you?

File: data/core/units/monsters/Fire_Dragon.cfg [hunk 1]
```diff
@@ -5,6 +5,7 @@
     race=monster
     # workaround to cut the transparent part when displayed in the help
     image="units/monsters/fire-dragon.png~CROP(3,0,154,154)"
+    profile="portraits/monsters/fire-dragon.png"
     [standing_anim]
         start_time=0
```
Comment (sevu) [inline data/core/units/monsters/Fire_Dragon.cfg]: Maybe let's move that 2 lines up, because the comment is about the cropping and the standing animation.

**Comment by MetaKnight** [comment]
Thanks for your efforts! 
While you're at it, what about Lomarfel's portrait that LordBob agreed for the Rider, and so the Outrider as well?

**Comment by nemaara** [comment]
I already replied to you about that on the corresponding issue; I don't think they're suitable to be added to the batch of elven portraits. They don't match stylistically what's already there. I'd be open to using the Rider portrait for someone else besides just the one in AOI (that is, for another campaign character or possibly a Rider alt), but the outrider portrait needs work to me.

**Comment by Gweddeoran** [comment]
I'm only talking about Lomarfel, LordBob is the one who suggested this, check the post on the forums.

**Comment by stevecotton** [comment]
@sevu or @nemaara , please would you revert aa714827bee "wmlinit rules for renaming orcish leader images"?
    
Reasons for reverting:
    
* Syntax error, it's missing a comma on the Dune Windrider line
* Spelling error, "ors" in the replacement file paths
* It didn't change an old rule for portraits/orcs/transparent/leader.png
   that mapped to the now-to-rename-from path (ditto for leader-2.png).

**Comment by sevu** [comment]
Thanks. I changed the mistakes.
@nemaara Thought about renaming grunt-6 to leader too? (Dropping the wmllint rule then)

**Comment by nemaara** [comment]
Should be fine to rename it, but it would have to be renamed in all instances where the portrait is used as well. Also might cause problems in some UMC, which is why I didn't rename it originally.

**Pull request merged**